### PR TITLE
Update implementation to allow going through full flow

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update implementation to allow going through full flow

Use LinkButton on CartPage.

Lowercase URL parameter.

Add temporary redirect from Payment page if After Sign flow.

Temporary comment out check if checkout is completed on confirmation page.

## Justify why they are needed

It's time to get the POC together!

This is an attempt to enable us to show off as much as possible of the new experience and share it with the rest of the org.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #600** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/600" target="_blank">Graphite</a> 
    * **PR #601** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/601" target="_blank">Graphite</a> 
      * **PR #605** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/605" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/605?utm_source=stack-comment).

**Review by simonauner — APPROVED** [review]
LGTM!

I got very excited and wanted to test this, but didn't manage to add anything to my cart!

File: apps/store/src/pages/confirmation.tsx [hunk 1]
```diff
@@ -4,28 +4,32 @@ import { ConfirmationPage } from '@/components/ConfirmationPage/ConfirmationPage
 import { getMobilePlatform } from '@/components/ConfirmationPage/ConfirmationPage.helpers'
 import { ConfirmationPageProps } from '@/components/ConfirmationPage/ConfirmationPage.types'
 import { LayoutWithMenu } from '@/components/LayoutWithMenu/LayoutWithMenu'
-import { PageLink } from '@/lib/PageLink'
+import { normalizeLocale } from '@/lib/l10n/locales'
+// import { PageLink } from '@/lib/PageLink'
```
Comment (alebedev) [inline apps/store/src/pages/confirmation.tsx:8]: commented code

File: apps/store/src/pages/confirmation.tsx [hunk 2]
```diff
@@ -4,28 +4,32 @@ import { ConfirmationPage } from '@/components/ConfirmationPage/ConfirmationPage
...
-  if (!locale || locale === 'default') return { notFound: true }
+  const locale = normalizeLocale(rawLocale)
+  if (!isLocale(locale)) return { notFound: true }
 
   const apolloClient = initializeApollo()
 
   const [shopSession, globalStory] = await Promise.all([
     getCurrentShopSessionServerSide({ req, res, apolloClient }),
-    getGlobalStory({ locale }),
+    getGlobalStory({ locale: locale.toLocaleLowerCase() }),
```
[the remark this answers is not in the document]
Comment (alebedev) [reply apps/store/src/pages/confirmation.tsx:26]: Well, nextjs and Storyblok slugs expect lowercase, all other use cases expect mixed case.  Maybe we can make it service-level responsibility to convert from unknown case locale to what they need

**Review by alebedev — APPROVED** [review]

File: apps/store/src/pages/checkout/payment.tsx [hunk 3]
```diff
@@ -15,14 +19,20 @@ const NextCheckoutPaymentPageAdyen: NextPage<CheckoutPaymentPageAdyenProps> = (p
 export const getServerSideProps: GetServerSideProps<CheckoutPaymentPageAdyenProps> = async (
   context,
 ) => {
-  const { req, res, locale } = context
+  const { req, res, locale: rawLocale } = context
 
-  if (!locale || locale === 'default') return { notFound: true }
+  const locale = normalizeLocale(rawLocale)
+  if (!isLocale(locale)) return { notFound: true }
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/pages/checkout/payment.tsx:25]: good point, will add that

File: apps/store/src/pages/confirmation.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/pages/confirmation.tsx:26]: yeah let's discuss how we want to deal with this... we should be able to treat this without being case sensitive I think

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/605?utm_source=stack-comment).


Job Reference: KCwGiw4J2hbtZ1o0vlpv

**Pull request merged**